packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m

```diff
@@ -96,6 +96,20 @@ - (instancetype)initWithFrame:(CGRect)frame
                         inConfiguration:configuration];
 
     _webView = [[FLTWKWebView alloc] initWithFrame:frame configuration:configuration];
+
+    // Background color
+    NSNumber* backgroundColorNSNumber = args[@"backgroundColor"];
+    if ([backgroundColorNSNumber isKindOfClass:[NSNumber class]]) {
+      int backgroundColorInt = [backgroundColorNSNumber intValue];
+      UIColor* backgroundColor = [UIColor colorWithRed:(backgroundColorInt >> 16 & 0xff) / 255.f
+                                                 green:(backgroundColorInt >> 8 & 0xff) / 255.f
+                                                  blue:(backgroundColorInt & 0xff) / 255.f
+                                                 alpha:(backgroundColorInt >> 24 & 0xff) / 255.f];
+      _webView.opaque = NO;
+      _webView.backgroundColor = UIColor.clearColor;
+      _webView.scrollView.backgroundColor = backgroundColor;
```

Comment:
Non-opaque views have worse performance, so it's almost certainly worth special-casing this:
```
int alpha = backgroundColorInt >> 24 & 0xff;
UIColor* backgroundColor = [... alpha:(alpha / 255.0)];
if (alpha == 255) {
  _webView.backgroundColor = backgroundColor;
} else {
  [these three lines]
}
```

Reply:
I tried to make the change, but if I don't set `_webView.opaque = NO;` for an opaque color too the WebView's background is white on my emulator.

Reply:
That's unfortunate; hopefully that means the drawing for WKWebView is doing something non-obvious then which will avoid the performance penalty.